namegen.py

```diff
@@ -0,0 +1,35 @@
+import random
+def main():
+    first_names=('Lydia', 'Emma', 'Andrew', 'Josie', 'Owen', 'Jack', 'Tucker', 'Samantha', 'Frank',
+                'Bill', 'Joey', 'Bob', 'Gorg', 'Jose', 'Jorge', 'Scorge', 'Ben', 'Anita', 'Casey',
+                'Bella', 'Marcus', 'Austin', 'Mat', 'Grace', 'Mia', 'Connor', 'Harrison', 'Charlie',
+                'Taylor', 'Maxine', 'Anja', 'Aaron', 'Erin', 'Ruthie', 'Emily', 'Chloe', 'Matt', 'David',
+                'Kim', 'Cutter', 'Aidan', 'Riley', 'Rosalie', 'Haden', 'Georgina')
+
+    word = ('Lydia', 'Emma', 'Andrew', 'Josie', 'Owen', 'Jack', 'Tucker', 'Samantha', 'Frank',
+                 'Bill', 'Joey', 'Bob', 'Gorg', 'Jose', 'Jorge', 'Scorge', 'Ben', 'Anita', 'Casey',
+                 'Bella', 'Marcus', 'Austin', 'Mat', 'Grace', 'Mia', 'Connor', 'Harrison', 'Charlie',
+                 'Taylor', 'Maxine', 'Anja', 'Aaron', 'Erin', 'Ruthie', 'Emily', 'Chloe', 'Matt', 'David',
+                 'Kim', 'Cutter', 'Aidan', 'Riley', 'Rosalie', 'Haden', 'Georgina')
+first_names='word'[3]
```

Comment:
- [[PEP 8]](https://www.python.org/dev/peps/pep-0008/) It's recommended to have a space between operators such as `=`.

- `first_names` isn't the best name for this variable, since it actually only holds the value of one name, and it shadows `first_names` in main().

- [[PEP 8]](https://www.python.org/dev/peps/pep-0008/) I'd suggest placing two line breaks before line 14, as functions should be seperated with two line breaks.